cookbooks/aws-parallelcluster-environment/spec/unit/recipes/login_nodes_keys_spec.rb

```diff
@@ -1,33 +1,39 @@
 require 'spec_helper'
 
 describe 'aws-parallelcluster-environment::login_nodes_keys' do
+  SHARED_DIR_LOGIN_NODES = "/SHARED_DIR_LOGIN_NODES".freeze
```

Comment:
should this have #{} around it?

"/#{SHARED_DIR_LOGIN_NODES}"

Reply:
Why are you defining the shared dir again?  It is defined as an attribute in aws-parallelcluster-shared